### DISCUSSION TITLE
Should I "stop the interpreter" before closing the app?

### DISCUSSION THREAD

**Discussion by sebastian-blandon** [posit-dev/positron#4309] [Q&A]
Hi there!

Is clicking "stop the interpreter" necesary before closing Positron? I noticed that the app takes too much starting up after not stopping the interpreter before closing.

**Comment by jmcphers** [comment]
It should not be necessary to manually stop the interpreter! If you're still seeing this, could you check your system's task manager and let us know if there are any stray `python` or `ark` processes -- and if so, what their command-line arguments are?
